src/render/vertex_array_object.js

```diff
@@ -13,7 +13,7 @@ class VertexArrayObject {
     boundIndexBuffer: ?IndexBuffer;
     boundVertexOffset: ?number;
     boundDynamicVertexBuffer: ?VertexBuffer;
-    boundOpacityVertexBuffer: ?VertexBuffer;
+    boundDynamicVertexBuffer2: ?VertexBuffer;
     boundCollisionVertexBuffer: ?VertexBuffer;
```

Comment:
`boundCollisionVertexBuffer` should be removed here, right?

Reply:
👍